airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/build_image/steps/java_connectors.py

```diff
@@ -60,7 +60,8 @@ async def run_connector_build(context: ConnectorContext) -> StepResult:
     build_connector_tar_result = await BuildConnectorDistributionTar(context).run()
     if build_connector_tar_result.status is not StepStatus.SUCCESS:
         return build_connector_tar_result
-    dist_dir = await build_connector_tar_result.output.directory(dist_tar_directory_path(context))
+
+    dist_dir = await build_connector_tar_result.output.directory("build/distributions")
```

Comment:
for understanding: why did this change? feels unrelated to the other changes, maybe I'm missing something.

Reply:
It changes because the original path related to an absolute in container path. the `build_connector_tar` step now returns a directory which is the connector directory